sshsigner_test.go

```diff
@@ -480,22 +530,42 @@ func TestSSHSigner_SignHostKey(t *testing.T) {
 				verifier,
 				"aud",
 				tc.validAWSAccounts,
+				tc.opts...,
 			)
 			if err != nil {
 				t.Fatal(err)
 			}
+			signer.clock = func() time.Time { return now }
 
 			ctx := metadata.NewIncomingContext(
 				context.Background(),
 				metadata.New(map[string]string{"authorization": "bearer " + validToken}),
 			)
 
-			_, err = signer.SignHostKey(ctx, tc.req)
+			resp, err := signer.SignHostKey(ctx, tc.req)
 			if (tc.wantErr != "" && err == nil) || (tc.wantErr == "" && err != nil) {
 				t.Fatalf("wantErr: %v, err: %v", tc.wantErr, err)
 			} else if tc.wantErr != "" && !strings.Contains(err.Error(), tc.wantErr) {
 				t.Fatalf("wantErr: %v, err: %v", tc.wantErr, err)
 			}
+
+			if tc.wantErr == "" {
+				akey, _, _, _, err := ssh.ParseAuthorizedKey([]byte(resp.SignedCertificate))
+				if err != nil {
+					t.Fatal(err)
+				}
+
+				cert, ok := akey.(*ssh.Certificate)
+				if !ok {
+					t.Fatalf("wanted a %T, got a %T", &ssh.Certificate{}, cert)
+				}
+
+				fmt.Printf("%#v\n", cert)
```

Comment:
Debug statement left behind?

Reply:
copy & paste from before, removed both in 508b445